### PR TITLE
IPC Fixes

### PR DESCRIPTION
**Pull request by sMMiles**

### CODE REVIEW

File: src/sharding/clustermanager.js [hunk 1]
```diff
@@ -298,15 +298,15 @@ class ClusterManager extends EventEmitter {
                         break;
 
                     case "fetchUser":
-                        this.fetchInfo(1, "fetchUser", message.id);
```
Comment (ParadoxOrigins) [inline src/sharding/clustermanager.js]: don't worker IDs start at 1

Comment (sMMiles) [reply src/sharding/clustermanager.js]: Not anymore :mmLol:

Comment (ParadoxOrigins) [reply src/sharding/clustermanager.js]: i am pretty sure nodejs worker IDs start at 1

Comment (sMMiles) [reply src/sharding/clustermanager.js]: Not sure about that tbh but this fetches the worker object through the cluster ID which is then mapped to the worker ID anyway and we know for sure cluster ID starts at 0

Comment (ParadoxOrigins) [reply src/sharding/clustermanager.js]: you right actually this is fine, was reading the fetchInfo() code wrong

**Pull request merged**